### ISSUE TITLE
Tab Order incorrect

### ISSUE THREAD

**Issue by GoogleCodeExporter** [cryptowest/win-sshfs#105]
```
What steps will reproduce the problem?
1. Open sshfs manager.
2. Add new connection.
3.

What is the expected output? What do you see instead?
When creating a new connection, tab order should be: Drive Name, Host, Port, 
Username, Auth Method, Directory, Drive Letter, Mount at login, Password, Save, 
Mount.

Instead, it is: 
Drive Name, Host, Port, Username, Auth Method, Password, Directory, Drive 
Letter, Mount at login, Save, Mount.

What version of the product are you using?

win-sshfs: 0.0.1.5
Client OS: Win 8.1 x64
Server OS: N/A
Ssh server: Debian, but not applicable to this minor bug report.





Please provide any additional information below.


```

Original issue reported on code.google.com by `michael....@gmail.com` on 26 Aug 2014 at 4:13
